code/game/machinery/computer/crew.dm

```diff
@@ -98,40 +98,47 @@ GLOBAL_DATUM_INIT(crewmonitor, /datum/crewmonitor, new)
 		// Note that jobs divisible by 10 are considered heads of staff, and bolded
```

Comment:
what a horrible sorting system lmao

Reply:
Yeah, I totally agree, I had to comment some shit so our security stuff would fit 😔